packages/core/src/common/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const minimumSeverity = 3;
+export const testSound = 'https://s3-us-west-2.amazonaws.com/s.cdpn.io/858/outfoxing.mp3';
```

Comment:
Are these sounds used anywhere in this commit? If not we should leave it out for now.